lib/style/single_node.ex

```diff
@@ -139,6 +147,18 @@ defmodule Styler.Style.SingleNode do
   defp style({:case, cm, [head, [{do_, arrows}]]}), do: {:case, cm, [head, [{do_, rewrite_arrows(arrows)}]]}
   defp style({:fn, m, arrows}), do: {:fn, m, rewrite_arrows(arrows)}
 
+  # Negation of Enum.any? => Enum.empty? and Negation of Enum.empty? => Enum.any?
+  defp style({negator, _, [{{:., dm, [{:__aliases__, am, [:Enum]}, enum_function]}, funm, [arg]}]})
+       when enum_function in @enum_functions and is_negator(negator) do
+    {{:., dm, [{:__aliases__, am, [:Enum]}, Map.get(@enum_functions_mapper, enum_function)]}, funm, [arg]}
+  end
```

Comment:
i'd rather just have two function heads, one going each way, than rely on module attributes :) (also there are a lot of enum functions besides those two!)